app/src/main/java/ch/epfl/sweng/swenggolf/offer/ListOfferActivity.java

```diff
@@ -90,7 +91,7 @@ private void expandOrRetractOffer(TextView element, Offer offer) {
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstance) {
         setToolbar(R.drawable.ic_menu_black_24dp, R.string.offers);
-        View inflated = inflater.inflate(R.layout.activity_list_offer, container, false);
+        final View inflated = inflater.inflate(R.layout.activity_list_offer, container, false);
```

Comment:
why is the final useful here?

Reply:
You're right it's not absolutely necessary. But since inflated is never modified it just adds more information. Do you think I should remove it ?

Reply:
inflated is a method variable so I don't think it is very important, however if you think it's better we can leave the final :)

Reply:
Well, we leave it final then.